src/Entity/Transformer.php

```diff
@@ -194,6 +199,9 @@ public function loadBlock(array $content, PageInterface $page)
         $updatedAt->setTimestamp((int) $content['updated_at']);
         $block->setUpdatedAt($updatedAt);
 
+        /**
+         * @phpstan-var BlockContent $child
```

Comment:
Had to do this, because phpstan and psalm do not support nested type definition.